nerfstudio/process_data/process_data_utils.py

```diff
@@ -49,12 +49,16 @@ class CameraModel(Enum):
     OPENCV = "OPENCV"
     OPENCV_FISHEYE = "OPENCV_FISHEYE"
     EQUIRECTANGULAR = "EQUIRECTANGULAR"
+    PINHOLE = "PINHOLE"
+    SIMPLE_PINHOLE = "SIMPLE_PINHOLE"
 
 
 CAMERA_MODELS = {
     "perspective": CameraModel.OPENCV,
     "fisheye": CameraModel.OPENCV_FISHEYE,
     "equirectangular": CameraModel.EQUIRECTANGULAR,
+    "pinhole": CameraModel.PINHOLE,
+    "simple_pinhole" : CameraModel.SIMPLE_PINHOLE,
```

Comment:
Delete the extra space after `"simple_pinhole"` to make it `"simple_pinhole":` not `"simple_pinhole" :`. This will pass the Ruff check.